packages/remix-server-runtime/cookies.ts

```diff
@@ -31,7 +31,7 @@ export type CookieOptions = CookieParseOptions &
  *
  * @see https://remix.run/api/remix#cookie-api
  */
-export interface Cookie {
+export interface Cookie<Value = any> {
```

Comment:
In this case it doesn't really matter, but it's a good practice to say what type it should extend.

```suggestion
export interface Cookie<Value extends any = any> {
```